lib/tokenable/controllers/tokens_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Tokenable
+  class TokensController < ::ActionController::API
+    include Authable
+
+    def create
+      user_id, _ = User.from_params(params)
```

Comment:
@scottrobertson I added a second return param reserved for when we do revocations. We can use this to check against the jwt decoded content and see if it matches. Then we can do revocations by changing an attribute on the user object.